po/POTFILES

```diff
@@ -5,7 +5,6 @@ data/io.github.mightycreak.Diffuse.desktop.in
 data/io.github.mightycreak.Diffuse.appdata.xml.in
 src/diffuse/constants.py
 src/diffuse/dialogs.py
-src/diffuse/diffuse.in
```

Comment:
question: is there a specific reason why we should remove this file from the list?

I know it doesn't have any translation strings in it, but it could. If the reason is just because of the xgettext warning, then I might have found a better fix: rename the file to `diffuse.py.in`.

Reply:
I got a warning and while the file is not relevant to translation, then I removed it.
```
xgettext: warning: file 'src/diffuse/diffuse.in' extension '' is unknown; will try C
```